### PR TITLE
Fix wrong encodings

### PR DESCRIPTION
**Pull request by jnussbaum**

### CODE REVIEW

File: .github/workflows/automated-tests.yml [hunk 1]
```diff
@@ -14,9 +14,9 @@ jobs:
         runs-on: ubuntu-latest
         steps:
           - name: Checkout repo
-            uses: actions/checkout@v3
+            uses: actions/checkout@v4
```
Comment (Nora-Olivia-Ammann) [inline .github/workflows/automated-tests.yml:17]: What does this have to do with the rest?

**Review by Nora-Olivia-Ammann — APPROVED** [review]

File: .github/workflows/automated-tests.yml (hunk 1 above)
Comment (jnussbaum) [reply .github/workflows/automated-tests.yml:17]: I received a deprecation warning, and was asked to use a newer version of these 2 actions.

It is unrelated, so strictly speaking, I should have done another PR for it. But because it's such a small thing, I decided to make 1 PR for both.
